### PR TITLE
I367 Hide Coordination menu

### PR DESCRIPTION
**Pull request by psaavedra**
* Added the MENU_COORDINATION config parameter
* When MENU_COORDINATION is set to FALSE, the Coordination menu in
  the sidebar is hidden

Refers to #367

### CODE REVIEW

**Comment by jaragunde** [comment]
This is a very interesting approach, thanks for coding it. There are more things that we could hide when the coordination feature is disabled:
* The "task story" entry in the tasks screen.
* The "task story" entry in the user tasks report, and the corresponding column from the grid.

**Comment by jaragunde** [comment]
Ah, I would change the commit/PR title, because you aren't modifying the users evaluation and project summary entries.

**Comment by psaavedra** [comment]
Oops, please go ahead.  It seems a copy and paste matter.

File: web/js/tasks.js [hunk 1]
```diff
@@ -699,6 +699,43 @@ var TaskPanel = Ext.extend(Ext.Panel, {
...
+                    width: 186,
+                    layout: 'hbox',
+                    items:[
+                        this.deleteButton,
+                        this.cloneButton,
+                        this.createTemplateButton,
+                    ]
+                }),
+                new Ext.Container({
+                    width: 120,
```
Comment (jaragunde) [inline web/js/tasks.js]: Please use 145, like in the original code. That figure was tested with several browsers.
When the default font of the browser is slightly wider, the "onsite" text falls to the line below like this:
screenshot from 2017-09-04 12-08-47

File: web/js/tasks.js [hunk 2]
```diff
@@ -699,6 +699,43 @@ var TaskPanel = Ext.extend(Ext.Panel, {
...
+                    layout: 'hbox',
+                    items:[
+                        this.teleworkCheckBox,
+                        this.onsiteCheckBox
+                    ]
+                }),
+                new Ext.Container({
+                    width: 460,
```
Comment (jaragunde) [inline web/js/tasks.js]: We should probably use a different width here when coordination is disabled. With my usual browser and window size, it looks like this:
screenshot from 2017-09-04 12-08-47

File: web/js/tasks.js (hunk 1 above)
Comment (jaragunde) [reply web/js/tasks.js]: Also, check the indentation of this piece of code, it's mixing 8 and 4 spaces.

**Comment by psaavedra** [comment]
Reverted the *width* value modification. The issue in the visualization of the labels seems it went away too

**Pull request merged**